utilities/chemiscope.py

```diff
@@ -91,27 +91,21 @@ def chemiscope_input(meta, frames, projection, prediction,
 
     .. _`ase.Atoms`: https://wiki.fysik.dtu.dk/ase/ase/atoms.html
     '''
-
+    AUTHORIZED_KEYS = ["name", "description", "authors", "references"]
     data = {
         'meta': {
-            'name': meta['name'],
-            'description': meta.get('description', None),
-            'authors': meta.get('authors', None),
-            'references': meta.get('references', None),
+            key: value for key, value in meta.items() if key in AUTHORIZED_KEYS
```

Comment:
@Luthaf might it be better like this?

```suggestion
            key: meta.get(key, None) for key in AUTHORIZED_KEYS
```

- This will also reduce loop time if there are ~infinite keys in meta.
- Also might suggest `AUTHORIZED_KEYS` --> `REQUIRED_KEYS`

Reply:
no, the whole issue comes frome using `None`, which is translated into `null` by Python's json module, which is different from `undefined` on the JS side. 

> This will also reduce loop time if there are ~infinite keys in meta.

I think having a very large number of invalid keys in meta is not something we want to optimize for  =)

> Also might suggest AUTHORIZED_KEYS --> REQUIRED_KEYS

Only `name` is required, the other keys are not.

Reply:
Bah, I had so many plans for infinite keys in meta. @hurricane642 let's remove this change from this PR and circle back at another time.

Reply:
I provided the above code (`key: value for key, value in meta.items() if key in AUTHORIZED_KEYS`) to @hurricane642 to fix the issue he's been seeing with the previous code (https://github.com/cosmo-epfl/kernel-tutorials/pull/17#discussion_r497389919), so the current state of the code is the right one here 😃